packages/react-core/src/demos/DescriptionList/examples/DescriptionListDrawer.tsx

```diff
@@ -0,0 +1,166 @@
+import React from 'react';
+import {
+  Gallery,
+  PageSection,
+  GalleryItem,
+  Card,
+  CardBody,
+  Drawer,
+  DrawerPanelContent,
+  DrawerContent,
+  DrawerHead,
+  DrawerActions,
+  DrawerCloseButton,
+  PageSectionVariants,
+  TextContent,
+  Text,
+  DrawerPanelBody,
+  DescriptionList,
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+  Button,
+  Page,
+  Title
+} from '@patternfly/react-core';
+import { DashboardWrapper } from '@patternfly/react-core/dist/js/demos/DashboardWrapper';
+import PlusCircleIcon from '@patternfly/react-icons/dist/esm/icons/plus-circle-icon';
+
+export const DescriptionListDrawer: React.FunctionComponent = () => {
+  const drawerRef = React.useRef<HTMLDivElement>();
+  const [isExpanded, setIsExpanded] = React.useState(true);
+
+  const onExpand = () => {
+    drawerRef.current && drawerRef.current.focus();
+  };
+
+  const onCloseClick = () => {
+    setIsExpanded(false);
+  };
+
+  const onOpenDrawer = () => {
+    setIsExpanded(true);
+  };
```

Comment:
In addition to opening/closing the drawer, we should set focus on the appropriate element.

- For the Drawer onExpand, the close button of the Drawer should receive focus, similar to our [basic drawer example](https://patternfly-react-pr-10229.surge.sh/components/drawer#basic)
- When the Drawer gets closed, focus should be set back on the element that previously had focus before the Drawer was expanded. In this case it's easier since there's only the one button controlling expansion.

Reply:
Can you please help me on first point,  is there any props in drawer component for setting focus on close button. In basic drawer example, I can't see the focus on close button.

Second point I've covered.